src/Search/Console.php

```diff
@@ -343,26 +341,15 @@ public function rdsReindexCommand(InputInterface $input, OutputInterface $output
         $this->logger->info('RDS articles added to indexing queue.');
     }
 
-    public function reviewedPreprintReindexCommand(InputInterface $input, OutputInterface $output)
-    {
-        $this->logger->info('Reindex reviewed preprints...');
-        $ids = [];
-        foreach (array_keys($this->config['reviewed_preprints']) as $id) {
-            $this->logger->info("Queuing reviewed preprint article $id");
-            $this->enqueue('article', $id);
-            $this->logger->info("Queuing reviewed preprint $id");
-            $this->enqueue('reviewed-preprint', $id);
-            $ids[] = $id;
-        }
-        $output->writeln('Queued: '.implode(', ', $ids).' (reviewed-preprint and article)');
-        $this->logger->info('Reviewed preprints added to indexing queue.');
-    }
-
     public function reviewedPreprintPurgeCommand(InputInterface $input, OutputInterface $output)
     {
         $this->logger->info('Purge reviewed preprints...');
         $ids = [];
-        foreach (array_keys($this->config['reviewed_preprints']) as $id) {
+        /** @var ApiSdk $sdk */
+        $sdk = $this->kernel->get('api.sdk');
```

Comment:
Let's remove the need to call the api here and just purge all reviewed preprints from the elasticsearch. There may be a single command to do this or we can read them from the index and then delete each one in turn.